ios/RNPlayerViewManager.swift

```diff
@@ -71,6 +71,28 @@ class RNPlayerViewManager: RCTViewManager {
         self.customMessageHandlerBridgeId = customMessageHandlerBridgeId
     }
 
+     @objc func setFullscreen(_ viewId: NSNumber, isFullscreen: Bool) {
+         bridge.uiManager.addUIBlock { [weak self] _, views in
+             guard
+                 let self,
+                 let view = views?[viewId] as? RNPlayerView
+             else {
+                 return
+             }
+             guard let playerView = view.playerView else {
+                 return
+             }
+             if playerView.isFullscreen == isFullscreen {
+                 return
+             }
```

Comment:
`guard` would be the idiomatic way in Swift to return early in a situation like this.

```suggestion
             guard playerView.isFullscreen != isFullscreen else {
                 return
             }
```